servers/rendering/rendering_device_graph.h

```diff
@@ -45,6 +45,20 @@
 
 class RenderingDeviceGraph {
 public:
+	struct RaytracingListInstruction {
+		enum Type {
+			TYPE_NONE,
+			TYPE_BUILD_ACCELERATION_STRUCTURE,
+			TYPE_BIND_PIPELINE,
+			TYPE_BIND_UNIFORM_SET,
+			TYPE_SET_PUSH_CONSTANT,
+			TYPE_TRACE_RAYS,
+			TYPE_UNIFORM_SET_PREPARE_FOR_USE
```

Comment:
Add trailing comma to enum:

```suggestion
			TYPE_UNIFORM_SET_PREPARE_FOR_USE,
```